twittermap/app/controllers/TwitterMapApplication.scala

```diff
@@ -21,13 +21,15 @@ class TwitterMapApplication @Inject()(val wsClient: WSClient,
   val USCityDataPath: String = config.getString("us.city.path").getOrElse("/public/data/city.sample.json")
   val cloudberryRegisterURL: String = config.getString("cloudberry.register").get
   val cloudberryWS: String = config.getString("cloudberry.ws").get
+  val sentimentEnabled: Boolean = config.getBoolean("sentimentEnabled").get
```

Comment:
I saw the codacy gives two good suggestions. We should give it a default value in case the configuration file is messed up.

`config.getBoolean("sentimentEnabled").getOrElse(false)`.